header/store/store.go

```diff
@@ -18,26 +18,13 @@ import (
 
 var log = logging.Logger("header/store")
 
-// TODO(@Wondertan): Those values must be configurable and proper defaults should be set for
-// specific node type. (#709)
-var (
-	// DefaultStoreCacheSize defines the amount of max entries allowed in the Header Store cache.
-	DefaultStoreCacheSize = 4096
-	// DefaultIndexCacheSize defines the amount of max entries allowed in the Height to Hash index
-	// cache.
-	DefaultIndexCacheSize = 16384
-	// DefaultWriteBatchSize defines the size of the batched header write.
-	// Headers are written in batches not to thrash the underlying Datastore with writes.
-	DefaultWriteBatchSize = 2048
-)
-
 var (
 	// errStoppedStore is returned for attempted operations on a stopped store
 	errStoppedStore = errors.New("stopped store")
 )
 
 // store implements the Store interface for ExtendedHeaders over Datastore.
-type store struct {
+type Store struct {
```

Comment:
For the team. As per our sync with @vgonkivs, we need to have it public so that we can access Params and test that they are set correctly. It's not a big deal, but in return, we always know that the config is set correctly by the unit test, with FX indirections.